static/css/devhub/new-landing/base.less

```diff
@@ -65,6 +65,23 @@ img {
   max-width: 100%;
 }
 
+.link(@color, @decoration: none) {
+  color: @color;
+  text-decoration: @decoration;
+
+  &:visited {
+    color: fadeOut(@color, 10%);
+  }
+
+  &:hover {
```

Comment:
See also https://www.w3.org/TR/2008/NOTE-WCAG20-TECHS-20081211/G183 for more rationale.